src/imagej/util.py

```diff
@@ -0,0 +1,105 @@
+from itertools import cycle
+from shutil import get_terminal_size
+from threading import Thread
+from time import sleep
+from typing import List
+
+
```

Comment:
Probably a good idea to link the SO post in the code

Reply:
Added the link to the `Loader` class docstring (0df568fdf7ec8c81a6eab407e92009300184561b).